components/paragraph/src/link-renderer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Anchor from './atoms/anchor';
+
+// Support react-router links
+// https://github.com/rexxars/react-markdown/issues/29#issuecomment-231556543
+function LinkRenderer({ href, children }) {
+  return (
+    href.match(/^\//)
+      ? <Link to={href}>{children}</Link>
+      : <Anchor href={href}>{children}</Anchor>
+  );
+}
+
+LinkRenderer.propTypes = {
+  href: PropTypes.string,
+  children: PropTypes.string,
```

Comment:
`children` should be `PropTypes.node`

Reply:
I would rather it stayed as a string as I only want the component to support strings and not nested nodes. AFAIK Markdown should be treated as a string.